src/overview/contributions.md

```diff
@@ -54,6 +54,8 @@ When [retrieving contributions](../api-specification/contribution-controller/ret
 | `balanceLeft` | `integer($int32)` | The remaining token balance after retrieving contributions. |
 | `contributionsNotReturned` | `integer($int32)` | The number of contributions that have not been returned. |
 | `contributionsNotReturnedCost` | `integer($int64)` | The total cost of all the contributions that have not been returned but met the filter requirements, i.e., the amount of tokens that were not available on the user's balance at the time of the retrieval. |
+| `assetDefinitionId` | `string` | The unique identifier of a contribution record on the blockchain network. |
+| `sourcePeerId` | `string` | The unique identifier of the original source that contributed the data. May be different from the `peerID`. <br> For details, see [Tokenomics: Reward split](./tokenomics.md#reward-split). |
```

Comment:
this should be added as an option attribute to the contribution assembling example/structure

Reply:
also I'd suggest expanding the description of the `Submitting contributions` with information about source peer id and that their balance would be increased based on the configured reward distribution (between uploader and source) ratio